product_info.toml

```diff
@@ -1,36 +1,36 @@
 title = "All known products that require license acceptance, and the mapping of parent to child relationships"
 
 [[products]]
-name = "chef-infra"
-pretty_name = "Chef Infra"
+id = "infra-client"
```

Comment:
@markan @tas50 This will require changing the string we provide to `LicenseAcceptance::Acceptor.check_and_persist`. I figured it was okay to do this since none of the products who use this call (excluding InSpec) have been released yet, and InSpec is not changing.